packages/docusaurus-theme-common/src/components/Details/styles.module.css

```diff
@@ -18,25 +18,23 @@ CSS variables, meant to be overriden by final theme
   position: relative;
   cursor: pointer;
   list-style: none;
-  margin-left: 1.8rem;
+  padding-left: 1rem;
 }
 
-.details > summary::-webkit-details-marker {
+.details > summary:marker {
```

Comment:
It looks like we can remove this CSS declaration altogether, since [from v89 Chrome respects `list-style:none`](https://groups.google.com/a/chromium.org/g/chromium-discuss/c/uXcmWMm2gRQ?pli=1) on `summary` element. Given that Google Chrome auto-updates, should we really remove that code?

Reply:
WebKit is the engine for Safari/iOS, so it's worth checking what's the Safari behavior too.

Reply:
I see, didn't take this into account, so revert it and added TODO comment.